### PR TITLE
[FIX][12.0] website_js_below_the_fold travis warning

### PR DESCRIPTION
**Pull request by tbaden**
issue https://github.com/OCA/website/issues/598 :
fix of:
************* Module website_js_below_the_fold
website_js_below_the_fold/readme/DESCRIPTION.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
website_js_below_the_fold/readme/CONTRIBUTORS.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
website_js_below_the_fold/readme/USAGE.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline

### CODE REVIEW

File: website_js_below_the_fold/readme/USAGE.rst [hunk 1]
```diff
@@ -2,4 +2,4 @@ This is a technical module and have no graphical interface whatsoever.
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline website_js_below_the_fold/readme/USAGE.rst]: This should be deleted from README by fragments. It's already in the header of the generated README.

Comment (tbaden) [reply website_js_below_the_fold/readme/USAGE.rst]: done

File: website_js_below_the_fold/readme/USAGE.rst [hunk 2]
```diff
@@ -1,5 +0,0 @@
-This is a technical module and have no graphical interface whatsoever.
```
Comment (pedrobaeza) [inline website_js_below_the_fold/readme/USAGE.rst]: Don't remove this text. Only the runbot part.

Comment (tbaden) [reply website_js_below_the_fold/readme/USAGE.rst]: ok, misunderstood this one. fixed now.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**